source/component.ts

```diff
@@ -202,7 +202,7 @@ export class Component {
 				[path]: component
 			},
 
-			route(path: string, component: RouteGroup): RouteableRouteGroup {
+			route(path: string, component: RouteGroup) {
```

Comment:
is this cleanup or to fix a typing issue?

Reply:
typing issue